### DISCUSSION TITLE
YARN doesn't install the template properly

### DISCUSSION THREAD

**Discussion by petr-hybler** [primefaces/community#1396] [PrimeVue Templates]
Hello,
I am used to work with YARN but it doesn't install the project properly (issues with layouts) ... however, when using NPM it alerts with some audit issues  ... that's not a really a good thing to have .. any idea why YARN is not working?

**Comment by tugcekucukoglu** [comment]
Hello,

How can we replicate the issue?

**Reply by petr-hybler** [reply]
just download the repo and use YARN instead of NPM :) 

Here is my part:

I downloaded apollo .. went to apollo-vite 

executed: yarn

`$ yarn
yarn install v1.22.19
info No lockfile found.
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
[1/4] Resolving packages...
[2/4] Fetching packages...
warning chart.js@4.4.2: The engine "pnpm" appears to be invalid.
[3/4] Linking dependencies...
warning " > @babel/eslint-parser@7.23.10" has unmet peer dependency "@babel/core@^7.11.0".
[4/4] Building fresh packages...
success Saved lockfile.
Done in 16.94s.`

Then 'yarn dev' to start it ... it started fine.. but the template is broken

![image](https://github.com/primefaces/community/assets/17429498/0422cfbb-d4f0-4ef5-bddb-f88a13e76af5)

**Reply by petr-hybler** [reply]
btw it's not typescript friendly at all !! :(

**Comment by petr-hybler** [comment]
any update?

**Comment by petr-hybler-ofti** [comment]
hello?

**Comment by petr-hybler-ofti** [comment]
hello?
